src/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -471,4 +472,15 @@ public enum ChannelBindingKind
         Unique = 25,
         Unknown = 0,
     }
+    public sealed class TokenBinding
+    {
+        public TokenBinding(TokenBindingType bindingType, byte[] rawTokenBindingId) { }
```

Comment:
@stephentoub I tried adding ifdefs etc as you described, but it still fails.  I have a feeling there's something I still haven't wired up.

The ref/System.Net.Primitives.builds didn't exist, so I created it based on the one you linked.  Do I have to point to this somewhere else in order to make it all work?